spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -533,7 +534,15 @@ public record ChatCompletionChunk(
 		public record ChunkChoice(
 				@JsonProperty("finish_reason") ChatCompletionFinishReason finishReason,
 				@JsonProperty("index") Integer index,
-				@JsonProperty("delta") ChatCompletionMessage delta) {
+				@JsonProperty("delta") Delta delta
+		) {
+			// The delta used in ChatCompletionChunk is not identical to ChatCompletion's ChatCompletionMessage, so a Delta record needs to be defined separately.
+			public record Delta(
```

Comment:
I'm afraid that this is incorrect. The message format is the same in both the [chat completion object](https://platform.openai.com/docs/api-reference/chat/object) and the [chat completion chunk object](https://platform.openai.com/docs/api-reference/chat/streaming) and it includes the tool_calls hierarchy as well.

Reply:
I misunderstood. Thank you for letting me know.